app/services/application_state_change.rb

```diff
@@ -16,7 +16,7 @@ class ApplicationStateChange
   REAPPLY_STATUSES = %i[rejected cancelled withdrawn declined offer_withdrawn].freeze
 
   TERMINAL_STATES = UNSUCCESSFUL_STATES + %i[recruited].freeze
-  IN_PROGRESS_STATES = DECISION_PENDING_STATUSES + ACCEPTED_STATES + %i[offer].freeze
+  IN_PROGRESS_STATES = DECISION_PENDING_STATUSES + ACCEPTED_STATES + %i[offer].freeze - %i[conditions_not_met]
```

Comment:
What is the reasoning behind not removing `condition_not_met` from the `ACCEPTED_STATES`? Do we think that'll cause more problems?

Reply:
Happy to create another card for this 👌

Reply:
Thank you!